app/models/division.rb

```diff
@@ -44,30 +38,56 @@ def acceptable_years_in_school
 
   private
```

Comment:
Since the code isn't reused by any other model and the division model is not too unwieldy I would say it might sacrifice some readability for newcomers to the project. If the file does get larger and start to contain repeated functionality I would be fine with putting that in a module.